acceptance/testdata/restorer/container/cnb/buildpacks/cacher_buildpack/cacher_v1/bin/build

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```

Comment:
If we're generating the tar on the fly we could get rid of everything in `acceptance/testdata/restorer/container/cnb/buildpacks` as it's not required for the tests to run successfully.

Reply:
I removed all the unused files now

Reply:
It looks like the tests all still pass if the following files are removed:

* acceptance/testdata/restorer/container/cnb/orders/cache_creator_order.toml
* acceptance/testdata/restorer/container/cnb/orders/cache_restorer_order.toml
* acceptance/testdata/restorer/container/layers/cacher_buildpack/uncached-layer.sha
* acceptance/testdata/restorer/container/layers/cacher_buildpack/uncached-layer.toml

Reply:
Yeah, I removed the orders. Good spot.